serverAkkaHttp/src/main/scala/RouterDerivation.scala

```diff
@@ -44,19 +43,17 @@ object RouterDerivationMacro extends RouterDerivationModule {
     """
   }
 
-  def deriveRouterImplPrinter[A: c.WeakTypeTag](c: Context)(a: c.Expr[A], printer: c.Expr[Printer]): c.Tree = {
+  def deriveRouterImplPrinter[A: c.WeakTypeTag](c: Context)(a: c.Tree, printer: c.Tree): c.Tree = {
     import c.universe._
     val tpe = weakTypeOf[A]
-    val derivePath = new MacroHelper[c.universe.type](c.universe).derivePath(tpe)
+    val derivePath = MacroHelper.derivePath(c.universe)(tpe)
 
     q"""
-    import wiro.{ OperationType, MethodMetaData }
-
-    new Router {
+    new _root_.wiro.server.akkaHttp.Router {
       override val routes = route[$tpe]($a)
       override val methodsMetaData = deriveMetaData[$tpe]
       override val tp = typePath[$tpe]
-      override implicit val printer = $printer
+      override implicit val printer: _root_.io.circe.Printer = $printer
```

Comment:
this is not strictly necessary for hygenic expansions, but annotating the implicit return type is generally a good idea.